templates/single-meetings.php

```diff
@@ -159,14 +159,23 @@ function tsml_body_class($classes) {
 											<p class="group-phone">
 												<a href="tel:<?php echo $meeting->phone ?>"><?php echo $meeting->phone ?></a>
 											</p>
-										</a>
 										<?php }
 										if (!empty($meeting->venmo)) {?>
-											<p class="group-venmo">
+											<p class="venmo">
 												Venmo: <a href="https://venmo.com/<?php echo substr($meeting->venmo, 1) ?>" target="_blank"><?php echo $meeting->venmo ?></a>
 											</p>
-										</a>
-										<?php }?>
+										<?php }
+										if (!empty($meeting->square)) {?>
+											<p class="square">
+												Square Cash: <a href="https://cash.app/<?php echo $meeting->square ?>" target="_blank"><?php echo $meeting->square ?></a>
+											</p>
+										<?php }
+										if (!empty($meeting->paypal)) {?>
+											<p class="paypal">
+												PayPal: <a href="<?php echo $meeting->paypal ?>" target="_blank"><?php echo substr($meeting->paypal, 22) ?></a>
```

Comment:
This "22" also seems to be relying on the PayPal.me URL. If we're sticking with it, can we add a comment to explain the "22" magic number?

Reply:
At this time neither the `PayPal` URL nor the username get are shown here.

Reply:
Yep good catch, this line needs to be fixed